### DISCUSSION TITLE
Combine two ggml_tensor like torch.cat

### DISCUSSION THREAD

**Discussion by sankalpdayal** [ggml-org/llama.cpp#1987] [Q&A] [answered]
I want to combine two embedding tensors where one of the embedding tensor is pre-stored which I plan to read from file. Is there a function 
1. Creates a ggm_tensor from two ggm_tensors of same dimensionality.
2. Loads a ggml tensors from lets say numpy object. Even csv should do for now.

**Comment by ggerganov** [comment, marked as answer]
This is how you can implement `torch.cat` with existing `ggml` operators:

https://github.com/ggerganov/ggml/blob/80f21b878163608209604a11083557a791342cf0/examples/sam/main.cpp#L1066-L1071
